packages/shopping/package.json

```diff
@@ -40,17 +40,20 @@
     "src"
   ],
   "dependencies": {
-    "@loopback/authentication": "3.2.1",
-    "@loopback/boot": "1.5.10",
-    "@loopback/context": "1.23.4",
-    "@loopback/core": "1.10.6",
-    "@loopback/openapi-v3": "1.10.0",
-    "@loopback/repository": "1.15.3",
-    "@loopback/rest": "1.22.0",
-    "@loopback/rest-explorer": "1.4.3",
-    "@loopback/service-proxy": "1.3.10",
-    "@types/jsonwebtoken": "8.3.5",
+    "@loopback/authentication": "^3.2.1",
+    "@loopback/authorization": "^0.4.3",
+    "@loopback/boot": "^1.5.10",
+    "@loopback/context": "^1.23.4",
+    "@loopback/core": "^1.10.6",
+    "@loopback/openapi-v3": "1.9.1",
+    "@loopback/repository": "1.10.1",
+    "@loopback/rest": "1.16.6",
+    "@loopback/rest-explorer": "1.3.1",
+    "@loopback/security": "^0.1.6",
```

Comment:
Please remove all leading `^` for version ranges.

Reply:
@deepakrkris ^ +1 and please also resolve the conflicts for 2 package-lock.json files. LGTM after cleaning up the package.json file :shipit:

Reply:
agreed